src/modules/Collection.js

```diff
@@ -0,0 +1,48 @@
+export default class Collection {
+  constructor(localStorageItem, items) {
+    this.localStorageItem = localStorageItem;
+    this.items = items || [];
+  }
+
+  addItem = (description) => {
+    const completed = false;
+    const index = this.items.length + 1;
+    this.items.push({
+      description,
+      completed,
+      index,
+    });
+    this.items.sort((a, b) => a.index - b.index);
+    localStorage.setItem(this.localStorageItem, JSON.stringify(this.items));
+  };
+
+  removeItem = (index) => {
+    const arrayIndexToDelete = index - 1;
+    this.items.splice(arrayIndexToDelete, 1);
+    const newItems = this.items.map((item, index) => ({ ...item, index: index + 1 }));
+    this.items = newItems;
+    this.items.sort((a, b) => a.index - b.index);
+    localStorage.setItem(this.localStorageItem, JSON.stringify(this.items));
```

Comment:
You can kindly refactor these lines in a function to DRY.

Reply:
Thanks! Now it's corrected using a function.